uswds/uswds-site
# USWDS-Site - About: Move release notes section to What’s New

# Summary

Moved release notes section into the what’s new page and adds a note about subscribing to releases on GitHub. This PR also removes the two column layout from the What’s New page to accommodate for the new section.

## Related issue

Closes #2136

## Preview link

[What’s new →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-release-notes-update/about/whats-new/)

## Problem statement

The current Release Notes page improperly displays release notes that are pulled in from GitHub.

## Solution

Since this is a low traffic page, we’ve opted to move release notes information to the What’s New page and directly link out to GitHub release notes.

From our [Office Hours meeting notes](https://docs.google.com/document/d/1zveLSehSuFG_mJdSgpPhRm2U8DdZPZ3kToVpy4rqzSA/edit#heading=h.4k11bj3r2nwk) (Google Docs 🔒):

> **Conclusion:** Remove release notes page. Forward the release notes url to “what’s new” page. Add a release notes section to the what’s new page. In that section, add link to release notes in github and info about subscribing to releases.
> 

## Testing and review

1. Visit [What’s New](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-release-notes-update/about/whats-new/) page.
2. Confirm content appropriately informs users of where to find release notes as well as how to subscribe to releases.
3. Check grammar, punctuation, and spelling.
4. Approve of page layout.
5. Inspect [community](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-release-notes-update/about/community/) page.
6. Confirm external link buttons have launch icon and appropriate margins.
7. Confirm “Release notes” page is removed from site navigation.
8. Confirm there are no links to the release notes page.

### Testing checklist

- [ ]  Information on release notes is properly conveyed.
- [ ]  Information on how to subscribe to releases is clear and easy to follow.
- [ ]  What’s New page is well presented.
- [ ]  Prose descendant external links styled as buttons have external link icon.
- [ ]  Standard external links unaffected by this change.
- [ ]  Internal links styled as buttons unaffected by this change.
- [ ]  No grammatical or spelling errors.
- [ ]  Expected redirects are present and working properly.
- [ ]  HTMLProofer runs without error.
